xbmc/utils/BitstreamConverter.h

```diff
@@ -87,8 +87,8 @@ class CBitstreamParser
   CBitstreamParser();
   ~CBitstreamParser();
 
-  static bool Open();
-  static void Close();
+  static bool Open(){};
```

Comment:
@peak3d You forgot to return. If called, this is undefined behaviour. It's a good idea enable -Werror=return-type to catch these types of errors..